extensions/amp-iframe/0.1/amp-iframe.js

```diff
@@ -46,7 +46,7 @@ export class AmpIframe extends AMP.BaseElement {
         this.element);
     const containerUrl = parseUrl(containerSrc);
     assert(
-        !((' ' + sandbox + ' ').match(/\s+allow-same-origin\s+/)) ||
+        !((' ' + this.sandbox_ + ' ').match(/\s+allow-same-origin\s+/)) ||
```

Comment:
It is lower cased above, but that must be at least proven by a test and probably better to just redo locally here.